### PR TITLE
HID-2152: remove outdated/legacy domains from allow-list

### PR DESCRIPTION
**Pull request by rupl**
# HID-2152

Just one step of the process. Not urgent, does not need to be in sync with a particular deploy.

### CODE REVIEW

File: api/services/HelperService.js [hunk 1]
```diff
@@ -8,36 +8,28 @@ const queryOptions = [
...
+
+  // Dev
+  'https://dev.api-humanitarian-id.ahconu.org',
+  'https://dev.auth-humanitarian-id.ahconu.org',
 
   // Local
-  'https://app.hid.vm',
   'https://api.hid.vm',
-  'http://app.hid.vm',
+  'https://auth.hid.vm',
```
Comment (teodorescuserban) [inline api/services/HelperService.js:30]:
```suggestion
  'http://auth.hid.vm',
```

**Review by teodorescuserban — APPROVED** [review]
with http on local i think it looks good.

File: api/services/HelperService.js (hunk 1 above)
Comment (rupl) [reply api/services/HelperService.js:30]: I have all four listed already...

File: api/services/HelperService.js [hunk 2]
```diff
@@ -8,36 +8,28 @@ const queryOptions = [
...
   'http://api.hid.vm',
-  'http://v3.hid.vm',
-  'http://admin.hid.vm',
-  'http://hid.vm',
+  'http://auth.hid.vm',
```
Comment (rupl) [inline api/services/HelperService.js:32]: @teodorescuserban

**Pull request merged**